lib/dry/monitor/rack/logger.rb

```diff
@@ -91,17 +91,17 @@ def filter_backtrace(backtrace)
         end
 
         def filter_params(params)
-          params.each_with_object({}) do |(k, v), h|
+          params.each do |k, v|
```

Comment:
@davydovanton can you measure the same but with this? `dup_params = params.dup; dup_params.each...` I know each_with_object is a little bit slower, but If you have time and you want, it would be nice to see the profiling with this.

Reply:
yep, I'll do it!

Reply:
so, I tried your idea. And the result is here:

```
Total allocated: 1462336 bytes (20615 objects)
Total retained:  368 bytes (2 objects)

allocated memory by gem
-----------------------------------
    672936  rack-2.0.4
    573920  dry-monitor/lib
    144000  uri
     41600  dry-events/lib
     25800  logger
      4000  other
        80  dry-configurable-0.11.5

allocated memory by file
-----------------------------------
    672936  /Users/anton/.rvm/gems/ruby-2.7.0/gems/rack-2.0.4/lib/rack/query_parser.rb
    487532  /Users/anton/work/repositories/dry-rb/dry-monitor/lib/dry/monitor/rack/logger.rb
    144000  /Users/anton/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/uri/common.rb
     47400  /Users/anton/work/repositories/dry-rb/dry-monitor/lib/dry/monitor/logger.rb
     41600  /Users/anton/work/repositories/dry-rb/dry-events/lib/dry/events/event.rb
     33600  /Users/anton/work/repositories/dry-rb/dry-monitor/lib/dry/monitor/rack/middleware.rb
     25800  /Users/anton/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/logger.rb
      5388  /Users/anton/work/repositories/dry-rb/dry-monitor/lib/dry/monitor/notifications.rb
      4000  test.rb
        80  /Users/anton/.rvm/gems/ruby-2.7.0/gems/dry-configurable-0.11.5/lib/dry/configurable/config.rb
```

you can see the difference here:

```
# current
Total allocated: 1395136 bytes (20215 objects)

# dup
Total allocated: 1462336 bytes (20615 objects)
```

Reply:
Thanks @davydovanton ;)